app/build.gradle

```diff
@@ -84,10 +84,8 @@ dependencies {
     implementation 'androidx.preference:preference-ktx:1.2.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1'
 
-    //implementation 'com.github.ARK-Builders:ark-filepicker:b2bfa01ea7'
-
-    implementation 'com.github.ark-builders:ark-filepicker:b2bfa01ea7'
-    implementation 'space.taran:arklib:0.1.0-SNAPSHOT-7df9a4e581'
+    implementation 'dev.arkbuilders:arklib:0.3.1'
+    implementation 'com.github.ark-builders:ark-filepicker:ae2c37ea87'
```

Comment:
@mdrlzy We should setup GitHub Packages for `ark-filepicker`.

Reply:
https://github.com/ARK-Builders/ark-components-android/issues/34
https://github.com/ARK-Builders/ark-components-android/issues/35